content/docs/getting-started/backends.md

```diff
@@ -153,7 +157,13 @@ function PageContent() {
 
 ## Other backends
 
-Tina is fairly unopinionated about how it receives data to edit, or where that data is stored. The backend you choose depends on your project and the React meta-framework you may be using. Currently, there are numerous packages to support Git & GitHub workflows, but Tina is designed to potentially work with any data source. We have also made working prototypes to source data from [Strapi](/guides/nextjs/tina-with-strapi/overview) and Contentful (not yet documented).
+Tina is backend-agnostic, meaning that you choose where to store your data based on the project requirements and the React meta-framework you may be using.
+
+Currently, there are numerous packages to support Git & GitHub workflows, as well as integration with 3rd party headless CMS (prototypes).
```

Comment:
I'm hesitant to list these options out as were trying to limit our coverage of maintaining topic indices to the actual index pages themselves.